new-log-viewer/src/typings/worker.ts

```diff
@@ -64,8 +66,20 @@ type WorkerReqMap = {
         cursor: CursorType,
         decoderOptions?: DecoderOptionsType
     },
+    [WORKER_REQ_CODE.QUERY_LOG]: {
+        searchString: string,
+        isRegex: boolean,
+        isCaseSensitive: boolean,
+    },
 };
 
+type ChunkResultType = {
```

Comment:
For consistency, let's
1. write `interface ChunkResultType {}` instead of `type ChunkResultType = {}`.
2. use `,` instead of `;` to separate the fields.